File: metric_tool/test/resources/kotlin/lcam/LcamD.kt
package io.company.app.view.basket.kotlin.lcam

internal class LcamD {
    private lateinit var myField: String
    private lateinit var anotherField: String

    fun setField(value: String) {
        myField = value
    }

    fun setAnotherField(value: String) {
        anotherField = value
    }

    fun getCombinedFields(): String {
        return myField + anotherField
    }
}

File: metric_tool/test/resources/dart/lcam/lcam_d.dart
library mylib;

class LcamD {
  final String _myField = "myField";
  final String _anotherField = "anotherField";

  String getField() {
    return _myField;
  }

  String getAnotherField() {
    return _anotherField;
  }

  String getCombinedFields() {
    return _myField + _anotherField;
  }
}
